app/src/main/java/org/oppia/app/testing/NavigationDrawerTestActivity.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.app.testing
+
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.home.HomeActivityPresenter
+import org.oppia.app.home.RouteToTopicListener
+import org.oppia.app.topic.TopicActivity
+import javax.inject.Inject
+
+class NavigationDrawerTestActivity : InjectableAppCompatActivity(), RouteToTopicListener {
+  @Inject
+  lateinit var homeActivityPresenter: HomeActivityPresenter
```

Comment:
Nit: I believe this line is broken into two lines after `Option + Cmd + L`, we need to shift this back to single line:

`@Inject lateinit var homeActivityPresenter: HomeActivityPresenter`

Reply:
Ditto in other files too.

Reply:
Most of our file are following the style suggested by Option + Cmd + L, so iam not updating this suggestion here... If this is a really important change then i think we need to create a separate PR so that all files follow same style.

Reply:
Yeah, even my files had similar issues because of the reformatting shortcut. But it has been suggested by @BenHenning in some of my PRs.
Also, please create an issue to track this.

Reply:
Okay will do that.

Reply:
@nikitamarysolomanpvt Please link that issue here. Also, haven't seen that issue been created yet, am I missing something here?

Reply:
https://github.com/oppia/oppia-android/issues/568#issue-540239711